website/docs/r/db2_instance.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+subcategory: "Db2 SaaS"
+layout: "ibm"
+page_title: "IBM : ibm_db2"
+description: |-
+  Manages IBM Db2 SaaS instance.
+---
+
+# ibm_db2
+
+Create or delete an IBM Cloud Db2 SaaS instance. The `ibmcloud_api_key` that are used by Terraform should grant IAM rights to create and modify IBM Cloud Db2 Databases and have access to the resource group the Db2 SaaS instance is associated with. For more information, see [documentation](https://cloud.ibm.com/docs/Db2onCloud?topic=Db2onCloud-getting-started) to manage Db2 SaaS instances.
+
+
+Configuration of an Db2 SaaS resource requires that the `region` parameter is set for the IBM provider in the `provider.tf` to be the same as the target Db2 SaaS `location/region`. If the Terraform configuration needs to deploy resources into multiple regions, provider alias can be used. For more information, see [Terraform provider configuration](https://www.terraform.io/docs/configuration/providers.html#multiple-provider-instances).
+
+
+## Example usage
+To find an example for provisioning and configuring a Db2 SaaS instance , see [here](https://github.ibm.com/Akash-David/terraform-provider-db2oc/tree/main/examples/ibm-db2).
```

Comment:
Change link to: https://github.com/IBM-Cloud/terraform-provider-ibm/tree/master/examples/ibm-db2